backend/onedrive/onedrive.go

```diff
@@ -1818,7 +1905,22 @@ func newOptsCall(normalizedID string, method string, route string) (opts rest.Op
 // parseNormalizedID parses a normalized ID (may be in the form `driveID#itemID` or just `itemID`)
 // and returns itemID, driveID, rootURL.
 // Such a normalized ID can come from (*Item).GetID()
-func parseNormalizedID(ID string) (string, string, string) {
+func parseNormalizedID(ID string, Region string) (string, string, string) {
+	graphURL := graphAPIEndpoint + "/v1.0"
+	switch Region {
+	case "us":
+		graphURL = graphAPIEndpointUS + "/v1.0"
+		break
+	case "de":
+		graphURL = graphAPIEndpointUS + "/v1.0"
```

Comment:
```suggestion
		graphURL = graphAPIEndpointDE + "/v1.0"
```

Looks like a typo here. Consider using a map (same suggestion as above) to avoid these.